src/Lykke.Service.HFT.Services/MatchingEngineAdapter.cs

```diff
@@ -113,7 +113,10 @@ public class MatchingEngineAdapter : IMatchingEngineAdapter
             {
                 return ResponseModel<Guid>.CreateOk(requestId);
             }
-            return ConvertToApiModel<Guid>(response.Status);
+
+            var responseModel = ConvertToApiModel<Guid>(response.Status);
+            responseModel.Result = requestId;
```

Comment:
please add for this separate property in ResponseModel
Becouse Result - it's logical proiperrty for success responce
Thsi properrty (ResponseModel.RequestId) we can use in success responce too

Reply:
It is a temporary measure for returning order id even for failed orders. Contracts will be refactored in separate task.